src/main/java/com/cinchapi/common/collect/Sequences.java

```diff
@@ -27,6 +30,9 @@
 import com.google.common.collect.Streams;
 
 /**
+ * TODO: Refactor Sequences to safely wrap arrays/lists instead of
```

Comment:
a TODO should not be added as class level javadoc. Leave it as an inline comment within the class